src/utils/parseInputFile.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs';
+
+export const parseInputFile = (filename: string): string[][] => {
+  if (!filename) {
+    throw new Error('Filename is required');
+  }
+  const countryStrings: string[][] = [];
+  const lines = fs
+    .readFileSync(filename)
+    .toString()
+    .split('\n')
+    .map((line) => line.replace('\r', ''));
```

Comment:
Simplify process of splitting input file by new lines

In the code above we split by `\n` and after that do `map()` on each line in order to remove `\r`. I think it's not efficient. That's why I propose to split by regex with pattern `/\r?\n/`.

```suggestion
    .split(/\r?\n/);
```